src/golive/steps.md

```diff
@@ -48,7 +47,9 @@ Sometimes it can take Let's Encrypt a couple of minutes to provision the certifi
 
 Configure your DNS provider to point your domain to your Platform.sh Master environment domain name.
 
-The way to do so will vary somewhat depending on your registrar, but nearly all registrars should allow you to set a CNAME.  Some will call it an Alias or similar alternate name, but either way the intent is to say "this domain should always resolve to... this other domain".  Add a CNAME record from your desired domain (`www.example.com`) to the master environment hostname you wrote down earlier.
+The way to do so will vary somewhat depending on your registrar, but nearly all registrars should allow you to set a CNAME.  Some will call it an Alias or similar alternate name, but either way the intent is to say "this domain should always resolve to... this other domain".  
+
+You can access the CNAME target by running `platform environment:info` and looking for the `edge_hostname` property.  That is the host name by which Platform.sh knows your environment.  Add a CNAME record from your desired domain (`www.example.com`) to the value of the `edge_hostname`.
```

Comment:
Probably mention `platform environment:info edge_hostname`? Also, probably mention that it should be on the `master` environment?